src/Type/Row.purs

```diff
@@ -0,0 +1,27 @@
+module Type.Row
+  ( class RowLacks
+  ) where
+
+-- Must not be exported
+foreign import data RowLacking :: Symbol -> Symbol -> Type
+
+class RowLacksF (key :: Symbol)
```

Comment:
Here's a quick counter example with a single use of the existing `RowLacksF`:

* given `forall t. RowLacks "k" (k :: t, k :: String) => Int`
* start with `row = (k :: t, k :: String)` and `key = "k"`
* then `snoc row key Anything` to get `composed = (k :: t, k :: String, k :: Anything)`
* then uncons at `key` and expect `Anything` with `RowCons key Anything rest composed`
* which will succeed because `t` can unify with `Anything`
* and `rest = (k :: String, k :: Anything)`